src/core/absorption_chiller.jl

```diff
@@ -50,11 +49,13 @@
 ```
 
 !!! note "Required inputs"
-    To model AbsorptionChiller, you must provide at least one of the following: (i) `thermal_consumption_hot_water_or_steam` from $(HOT_WATER_OR_STEAM), (ii) 
-    (ii), `chp_prime_mover` from $(PRIME_MOVERS),or (iii) all of the "custom inputs" defined below.
-    If prime_mover is provided, any missing value from the "custom inputs" will be populated from data/absorption_chiller/defaults.json, 
-    based on the `thermal_consumption_hot_water_or_steam` or `prime_mover`. boiler_type is "steam" if `prime_mover` is "combustion_turbine" 
-    and is "hot_water" for all other `prime_mover` types.
+    To model AbsorptionChiller, you must provide at least one of the following: 
+    (i) `thermal_consumption_hot_water_or_steam` from $(HOT_WATER_OR_STEAM), 
+    (ii) `chp_prime_mover` from $(PRIME_MOVERS), or 
+    (iii) all of the "custom inputs" defined below.
```

Comment:
@Bill-Becker Which inputs specifically are the "customer inputs"? Do you have thoughts on making this help text more clear?

Reply:
Good point on it not being clear. I'll make some changes in here related to this and then update the docstring.

Reply:
I didn't change any functionality, but I updated the docstring to better reflect the actual behavior in https://github.com/NREL/REopt.jl/pull/160/commits/d5f0ae6d0dbd2d0e51c7755d0ff7598ba29315d7. There are actually no required inputs, as the docstring indicated.